src/storage/src/hummock/compactor/iterator.rs

```diff
@@ -46,7 +46,7 @@ pub struct SstableStreamIterator {
     /// Iterates over the KV-pairs of the current block.
     block_iter: Option<BlockIterator>,
 
-    seek_block_idx: usize,
+    block_idx: usize,
```

Comment:
Please add some documentation for `SstableStreamIterator` and the `block_metas` fields to explain that it may be reading only a sub-set of blocks.